graphql/schema.graphql

```diff
@@ -689,25 +689,20 @@ type TestResult {
   status: String!
   baseStatus: String
   testFile: String!
-  displayTestName: String
   logs: TestLog!
   exitCode: Int
   startTime: Time
   duration: Float
   endTime: Time
   taskId: String
   execution: Int
-  logTestName: String
 }
 
 type TestLog {
   url: String
   urlRaw: String
   urlLobster: String
   lineNum: Int
-  htmlDisplayURL: String @deprecated(reason: "htmlDisplayURL deprecated, use url instead (EVG-15418)")
```

Comment:
We still query on this field https://github.com/evergreen-ci/spruce/blob/1606a6056fa661fb3922a123a3bfb3ac5ea9580d/src/gql/queries/get-task-tests.graphql 
Before merging this we should remove our dependence on it in spruce and ensure it is deployed.

Reply:
Yep! This can't be merged until EVG-15418 is deployed.